seatunnel-apis/seatunnel-api-spark/src/main/java/org/apache/seatunnel/spark/SparkEnvironment.java

```diff
@@ -81,6 +88,12 @@ public CheckResult checkConfig() {
         return CheckResult.success();
     }
 
+    @Override
+    public void registerPlugin(List<URL> pluginPaths) {
+        LOGGER.info("register plugins :" + pluginPaths);
+        //  this.sparkSession.conf().set("spark.jars",pluginPaths.stream().map(URL::getPath).collect(Collectors.joining(",")));
```

Comment:
Please remove the comment `//` here.

Reply:
> Please remove the comment `//` here.

Because we use `--jar` parmeter, so registerPlugin will be empty at now.

Reply:
If so, it's need to add a `TODO` here.